ops/ai-eng/contracts-test-maintenance/CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# Changelog
```

Comment:
Changelog might be annoying to maintain, probably easier to not have it honestly